SpatialGDK/Source/SpatialGDK/Private/Interop/WellKnownEntitySystem.cpp

```diff
@@ -181,4 +197,104 @@ void WellKnownEntitySystem::MaybeClaimSnapshotPartition()
 	}
 }
 
+void WellKnownEntitySystem::CreateServerWorkerEntity()
+{
+	RetryServerWorkerEntityCreation(NetDriver->PackageMap->AllocateEntityId(), 1);
+}
+
+// Creates an entity authoritative on this server worker, ensuring it will be able to receive updates for the GSM.
+void WellKnownEntitySystem::RetryServerWorkerEntityCreation(Worker_EntityId EntityId, int AttemptCounter)
+{
+	check(NetDriver != nullptr);
+
+	TArray<FWorkerComponentData> Components;
+	Components.Add(Position().CreateComponentData());
+	Components.Add(Metadata(FString::Format(TEXT("WorkerEntity:{0}"), { Connection->GetWorkerId() })).CreateComponentData());
+	Components.Add(ServerWorker(Connection->GetWorkerId(), false, Connection->GetWorkerSystemEntityId()).CreateServerWorkerData());
+
+	AuthorityDelegationMap DelegationMap;
+	DelegationMap.Add(SpatialConstants::GDK_KNOWN_ENTITY_AUTH_COMPONENT_SET_ID, EntityId);
+	Components.Add(AuthorityDelegation(DelegationMap).CreateComponentData());
+
+	check(NetDriver != nullptr);
```

Comment:
Is this a duplicate of a check above?

Reply:
ha, yeah, didn't read the code I was copy pasting as thoroughly as I should have 😄